jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -136,6 +138,208 @@ ecma_builtin_typedarray_prototype_length_getter (ecma_value_t this_arg) /**< thi
 } /* ecma_builtin_typedarray_prototype_length_getter */
 
 /**
+ * Type of routine.
+ */
+typedef enum
+{
+  TYPEDARRAY_ROUTINE_EVERY, /**< routine: every */
+  TYPEDARRAY_ROUTINE_SOME, /**< routine: some */
+  TYPEDARRAY_ROUTINE_FOREACH, /**< routine: forEach */
+  TYPEDARRAY_ROUTINE__COUNT /**< count of the mode */
+} typedarray_routine_mode;
+
+/**
+ * The common function for 'every', 'some' and 'forEach'
+ * because theuy have the similar implementation.
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+routines_for_every_some_foreach (ecma_value_t this_arg, /**< this argument */
```

Comment:
The function name must start with ecma_builtin_typedarray_ even if it is a static function to support all-in-one builds.

Perhaps ecma_builtin_typedarray_exec_routine ?